source/js/app.js

```diff
@@ -104,25 +104,39 @@ NotificationDocInstance.addListener();
 NotificationInstance.setup();
 SidebarInstance.applySidebar();
 
-
 // Dynamic Sidebars
 const DynamicSidebarInstance = new DynamicSidebar();
 DynamicSidebarInstance.applySidebar();
 
 const filter = new Filter();
 
-// Modal
-const ModalInstance = new Modal;
-ModalInstance.enableModals();
+// Initialize the modal instance
```

Comment:
@sebastianthulin This works, modals are re-initialised after adding elements to the DOM after a fetch this way.